File: tests/fixtures/sample.go
package main

import "fmt"

type Animal struct {
	Name string
}

func (a *Animal) Speak() string {
	return ""
}

type Dog struct {
	Animal
}

func (d *Dog) Speak() string {
	return "Woof"
}

func Greet(name string) string {
	return fmt.Sprintf("Hello, %s", name)
}

File: tests/fixtures/sample.js
import { readFile } from "fs";

class Animal {
  constructor(name) {
    this.name = name;
  }

  speak() {
    return "";
  }
}

class Dog extends Animal {
  speak() {
    return "Woof";
  }
}

function greet(name) {
  return `Hello, ${name}`;
}
